Sources/Workspace/Workspace+Configuration.swift

```diff
@@ -684,9 +684,13 @@ extension Workspace.Configuration {
                 return RegistryConfiguration()
             }
 
-            let data: Data = try fileSystem.readFileContents(self.path)
-            let decoder = JSONDecoder.makeWithDefaults()
-            return try decoder.decode(RegistryConfiguration.self, from: data)
+            do {
+                let data: Data = try fileSystem.readFileContents(self.path)
+                let decoder = JSONDecoder.makeWithDefaults()
+                return try decoder.decode(RegistryConfiguration.self, from: data)
```

Comment:
nit: we have a helper function that could make this slightly shorter
```suggestion
                let decoder = JSONDecoder.makeWithDefaults()
                return try decoder.decode(path: self.path, fileSystem: self.fileSystem, as: RegistryConfiguration.self)
```